io-engine/src/bdev/device.rs

```diff
@@ -504,6 +510,40 @@ impl BlockDeviceHandle for SpdkBlockDeviceHandle {
             source: Errno::ENXIO,
         })
     }
+    /// Flush the io in buffer to disk, for the Local Block Device.
+    async fn flush_io(&self) -> Result<u64, CoreError> {
+        let is_write_cache =
+            unsafe { spdk_bdev_has_write_cache(self.device.0.as_inner_ref()) };
+        if !is_write_cache {
+            info!("No Write Cache, No need of Flush");
```

Comment:
Maybe this should be a debug?
Can we also add more information about who self?

Reply:
> Maybe this should be a debug? Can we also add more information about who self?

Accepted. Added extra logging information in all error scenario for better tracing.